core/i18n/src/ITwinLocalization.ts

```diff
@@ -176,7 +172,7 @@ export class ITwinLocalization implements Localization {
       throw new Error("Translation key must map to a string, but the given options will result in an object");
     }
 
-    const en = this.i18next.getFixedT("en", namespace);
+    const en = this.i18next.getFixedT(["en", "en-US"], namespace);
```

Comment:
whats the reason for this change?

as i understand, the `en-US` gets added due to `detectOptions.order: ["querystring", "navigator", "htmlTag"]`
`en` is the default fallback we set, and all our locale files are for `en`

Reply:
I just thought it made more sense to also check `en-US` as a backup when specifically trying to `getEnglishString()`. 

`getFixedT("en", ...)` will only check for the localization files in `locales/en/`. `getFixedT(["en", "en-US"], ...)` will check in `locales/en/` first, then `locales/en-US` as a backup. If there's a reason we only want the former, then lmk and I'll revert it.

But yes, typically for `getLocalizedString`, the language is detected as `en-US` and we set `en` as the backup.

Reply:
tbh i never understood why we as a library gave preference to english, and `getEnglishString` was apart of the Localization interface.

I assumed its because we needed english strings for the keyins in core-frontend, but even then, the fn shouldve been optional in the interface imo

if we're adding en-us, why arent we adding the [rest](https://github.com/ladjs/i18n-locales/blob/master/index.json#L74-L87)? i just dont see a need for this

maybe @calebmshafer or @evelynpreslar-bentley can chime in

Reply:
Let's leave it as is for now.

The reason for english preference is due to key-ins and before we add en-US I'd want to consider if there's something else we should be doing instead.